app/jobs/rstuf/check_job.rb

```diff
@@ -0,0 +1,18 @@
+class Rstuf::CheckJob < Rstuf::ApplicationJob
+  RetryException = Class.new(StandardError)
+  FailureExecption = Class.new(StandardError)
+  retry_on RetryException, wait: :exponentially_longer, attempts: 10
+
+  queue_with_priority PRIORITIES.fetch(:push)
+
+  def perform(task_id)
+    case Rstuf::Client.task_status(task_id)
+    when "SUCCESS"
+      # no-op, all good
+    when "FAILURE"
+      raise FailureExecption, "rstuf failed"
```

Comment:
we use RSTUF, Rstuf and rstuf. Maybe we could agree on first two forms? :D

Reply:
yup, let's use `RSTUF` as a name for service and `Rstuf` for Ruby namespace